File: tests/machine/x/php/match_full.php
<?php
$x = 2;
$label = match($x) {
    1 => "one",
    2 => "two",
    3 => "three",
    default => "unknown",
};
echo $label, PHP_EOL;
$day = "sun";
$mood = match($day) {
    "mon" => "tired",
    "fri" => "excited",
    "sun" => "relaxed",
    default => "normal",
};
echo $mood, PHP_EOL;
$ok = true;
$status = match($ok) {
    true => "confirmed",
    false => "denied",
    default => null,
};
echo $status, PHP_EOL;
function classify($n) {
    return match($n) {
    0 => "zero",
    1 => "one",
    default => "many",
};
}
var_dump(classify(0));
var_dump(classify(5));
?>

File: tests/human/x/fs/match_full.fs
let x = 2

let label =
    match x with
    | 1 -> "one"
    | 2 -> "two"
    | 3 -> "three"
    | _ -> "unknown"

printfn "%s" label

let day = "sun"

let mood =
    match day with
    | "mon" -> "tired"
    | "fri" -> "excited"
    | "sun" -> "relaxed"
    | _ -> "normal"

printfn "%s" mood

let ok = true

let status = if ok then "confirmed" else "denied"
printfn "%s" status

let classify n =
    match n with
    | 0 -> "zero"
    | 1 -> "one"
    | _ -> "many"

printfn "%s" (classify 0)
printfn "%s" (classify 5)
